lib/signaling/v2/peerconnection.js

```diff
@@ -486,33 +487,91 @@ class PeerConnectionV2 extends StateMachine {
   }
 
   /**
-   * Updates scaleResolutionDownBy for encoding layers.
-   * @param {number} width
-   * @param {number} height
+   *
+   * @returns true if adaptive simulcast is effective.
+   */
+  _isAdaptiveSimulcastEnabled() {
```

Comment:
This can be a getter.
```suggestion
  get _isAdaptiveSimulcastEnabled() {
```